politeiawww/politeiawww.go

```diff
@@ -114,6 +115,9 @@ type politeiawww struct {
 	userPubkeys     map[string]string               // [pubkey][userid]
 	userPaywallPool map[uuid.UUID]paywallPoolMember // [userid][paywallPoolMember]
 	commentScores   map[string]int64                // [token+commentID]resultVotes
+
+	// cmsDb is only used during cmswww mode
+	cmsDb database.Database
```

Comment:
Would prefer initialism to be a consistent case.  So `cmsDB` instead of `cmsDb`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms